src/sagas/projects.js

```diff
@@ -107,11 +107,27 @@ function generateProjectKey() {
 
 export function* projectExported({payload: {exportType}}) {
   if (exportType === 'repo') {
-    const state = yield select();
-    yield call(saveCurrentProject, state);
+    yield* saveCurrentProject();
   }
 }
 
+function* saveCurrentProject() {
+  const state = yield select();
+  const userId = getCurrentUserId(state);
+  const currentProject = getCurrentProject(state);
```

Comment:
Arguably it would be cleaner to do:

```js
const userId = yield select(getCurrentUserId);
const currentProject = yield select(getCurrentProject);
```